charts/axosyslog/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: axosyslog
+description: AxoSyslog for Kubernetes
+type: application
+version: 0.8.1
+appVersion: "4.6.0"
```

Comment:
It's fine as it is. I'd like to just point out that the release pipeline could also fill out this value (to avoid manual errors).

Reply:
I've now added a local shell script to bump the syslog-ng version number, which should eventually become a release pipeline.